assets/src/blocks/Submenu/SubmenuBlock.js

```diff
@@ -17,6 +17,7 @@ export const registerSubmenuBlock = () => {
 
   registerBlockType(BLOCK_NAME, {
     title: 'Submenu',
```

Comment:
Maybe we should already rename the block from Submenu to Table of Contents here too? It's a bit confusing otherwise

Reply:
I agree! 👍

Reply:
Done: https://jira.greenpeace.org/browse/PLANET-7610